### PR TITLE
Use petab.v1

### PR DESCRIPTION
**Pull request by dweindl**
Be compatible with, and require petab>=0.4.0

### CODE REVIEW

File: petabtests/cases/v2.0.0/sbml/0011/0011.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 from inspect import cleandoc
 
 import pandas as pd
-from petab.C import *
+from petab.v1.C import *
```
Comment (dilpath) [inline petabtests/cases/v2.0.0/sbml/0011/0011.py:4]: Should this already import from `petab.v2`?

**Review by dilpath — APPROVED** [review]
:+1:

File: petabtests/cases/v2.0.0/sbml/0011/0011.py (hunk 1 above)
Comment (dweindl) [reply petabtests/cases/v2.0.0/sbml/0011/0011.py:4]: For now it's the same. Once they diverge more, we'll have to re-organize test generation here. I'd wait until then.

Comment (dweindl) [reply petabtests/cases/v2.0.0/sbml/0011/0011.py:4]: Well, I can also just update them now.

Comment (dweindl) [reply petabtests/cases/v2.0.0/sbml/0011/0011.py]: When things break to due missing `petab.v2.C.PREEQUILIBRATION_CONDITION_ID`, we'll know we have to update the tests...

Comment (dweindl) [reply petabtests/cases/v2.0.0/sbml/0011/0011.py]: Oops, there is no petab.v2.C yet

**Pull request merged**